package.json

```diff
@@ -21,7 +21,8 @@
       "import": "./lib/esm/index.js",
       "require": "./lib/commonjs/index.js"
     },
-    "./bundle.css": "./lib/bundle.css"
+    "./bundle.css": "./lib/bundle.css",
+    "./bundle-no-preflight.css": "./lib/bundle-no-preflight.css"
```

Comment:
maybe `bundle-no-tailwind` or `bundle-no-resets` I imagine if you don't use tailwind you probably don't know what preflight is

Reply:
updated to `bundle-no-resets`